docs/notifications.md

```diff
@@ -0,0 +1,96 @@
+# Notifications
+
+### Notification Types
+
+#### Package Releases
+
+Private Packagist will notify you every time we find one or more new versions for one of your packages. All tags, branches and commits with a composer.json file in the root directory are either new or modified versions.
```

Comment:
This sentence doesn't really make sense to me, "Tags [...] are either new or modified versions.", pretty sure that's only for new tags?

Reply:
If the source reference of a tag changes then we do update the version in Private Packagist and this will trigger a release notification for the tag.

Reply:
@glaubinix Sure then "New or changed tags". But a tag is not constantly a new or modified version, that sentence is just wrong.